api/v1/address.go

```diff
@@ -88,16 +88,19 @@ func (a *apiV1) handleCreateAddresses(c echo.Context) error {
 	var response []string
 	err = a.db.Transaction(func(tx *gorm.DB) error {
 		for _, address := range addresses {
-			address.TenantID = db.ID(GetTenantContext(c).TenantID)
+			var addrInput db.Address
+			addrInput.TenantID = db.ID(GetTenantContext(c).TenantID)
+			addrInput.AddressActorID = &address.AddressActorID
+			addrInput.AddressRobust = &address.AddressRobust
 
-			res := tx.Clauses(clause.OnConflict{DoNothing: true}).Create(&address)
+			res := tx.Clauses(clause.OnConflict{DoNothing: true}).Create(&addrInput)
```

Comment:
these changes are unrelated to the Broker stuff but I was getting errors hitting this API as I was setting up my local DB so I fixed it up here

Reply:
Interesting and looks find assuming you verified it worked. Did you make a rest request and see the row added to the db?

Reply:
yes exactly